FunKiiU.py

```diff
@@ -54,8 +54,8 @@
                     dest='patch_dlc', help='This will disable unlocking all DLC content')
 parser.add_argument('-nopatchdemo', action='store_false', default=True,
                     dest='patch_demo', help='This will disable patching the demo play limit')
-parser.add_argument('-all', action='store_true', default=False, dest='download_all',
-                    help='Downloads/gets tickets for EVERYTHING from the keyfile')
+parser.add_argument('-download_regions', nargs="?", default=['USA', 'EUR', 'JPN'], dest='download_regions',
```

Comment:
I think '-region' is better, easier, I don't like the underscore in the argument that the user has to type.
But only a minor issue!